modules/router-store/src/router_store_module.ts

```diff
@@ -155,18 +165,31 @@ export function routerReducer<T = RouterStateSnapshot>(
   ],
 })
 export class StoreRouterConnectingModule {
+  static forRoot(config?: StoreRouterConfig): ModuleWithProviders;
+  static forRoot(config: StoreRouterConfig = {}): ModuleWithProviders {
+    return {
+      ngModule: StoreRouterConnectingModule,
+      providers: [{ provide: _ROUTER_CONFIG, useValue: config }],
```

Comment:
This won't work in AoT because the actual value for `useValue` property won't get resolved. You'll need to have two tokens, one for internal use that you register using `useValue` and another you register with `useFactory` that uses the first token as a dependency. Then you'll inject the public token in the constructor.